metadata.rb

```diff
@@ -1,11 +1,17 @@
 # encoding: UTF-8
 name 'pg-multi'
-maintainer 'Christopher Coffey'
-maintainer_email 'christopher.coffey@rackspace.com'
+maintainer 'Rackspace'
+maintainer_email 'rackspace-cookbooks@rackspace.com'
 license 'Apache 2.0'
 description 'Postgresql Master/Slave Streaming Replication wrapper cookbook'
+source_url 'https://github.com/rackspace-cookbooks/pg-multi'
+issues_url 'https://github.com/rackspace-cookbooks/pg-multi/issues'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.1.2'
+version '1.0.0' # bump this AFTER release, not in a PR or before
```

Comment:
yes bumps in PR are required in major breaking change version bumps, that rule only applies on standard patch bumps